lib/api/v3/work_packages/schema/work_package_schema_representer.rb

```diff
@@ -91,6 +91,11 @@ def initialize(schema, self_link:, **context)
             { href: @base_schema_link } if @base_schema_link
           end
 
+          link :attachments do
+            next if represented.work_package.hide_attachments?
+            { href: nil }
```

Comment:
🟡 is setting a `nil` link here better then setting a schema property?

Reply:
Yes, I think both options are not ideal, but using `schema` caused issues.